src/DropDownMenu/DropDownMenu.js

```diff
@@ -111,6 +111,10 @@ class DropDownMenu extends Component {
      */
     disabled: PropTypes.bool,
     /**
+     * Overrides default icon component
+     */
+    iconButtonElement: PropTypes.element,
```

Comment:
I saw PropTypes.element in use (https://github.com/callemall/material-ui/blob/master/src/IconMenu/IconMenu.js#L41), but will do